mainwindow.cpp

```diff
@@ -307,8 +311,7 @@ void MainWindow::replaceTableModel(QAbstractTableModel *newTableModel)
     // TODO: Check argument.
 
     // Remove temporary file is exist
-    // TODO: should not I check pointers like ptr != NULL instead of just ptr?
-    // TODO: should not I assign NULL instead of 0?
```

Comment:
On Wed, May 13, 2015 at 06:55:30AM -0700, math07 wrote:
> > @@ -307,8 +311,7 @@ void MainWindow::replaceTableModel(QAbstractTableModel *newTableModel)
> >      // TODO: Check argument.
> >  
> >      // Remove temporary file is exist
> > -    // TODO: should not I check pointers like ptr != NULL instead of just ptr?
> > -    // TODO: should not I assign NULL instead of 0?
> 
> I think it's personal choice, feature-wise it does the same thing.

It is a question of coding style.

Thanks!

-- 
Regards,
Aleksey Cherepanov